app/Http/Controllers/Auth/LoginController.php

```diff
@@ -41,4 +37,37 @@ public function __construct()
     {
         $this->middleware('guest', ['except' => 'logout']);
     }
+
```

Comment:
Any time I take a method and reproduce it purely to change its syntax, I like to A) pass through as much of the functionality up to its parent as I can, and B) note that it's 98% (or whatever) the same as its parent so that, in the future, someone knows why we have all this complexity here that aren't in most apps. E.g. "This is almost the exact same method as Illuminate\Whatever\LoginControllerTraitOrWhatever@login, just modified to accept a form request" (or whatever.)

Same note for the register method on the RegisterController.

Reply:
I agree. I've just changed to extend the parent methods and pass a RegisterFormRequest and LoginFormRequest. Hope this solves this issue.

Reply:
Brilliant--thanks!